magefile.go

```diff
@@ -895,6 +895,7 @@ func movePackagesToArchive(dropPath string, requiredPackages []string) string {
 			}
 
 			targetPath := filepath.Join(archivePath, rp)
+			os.MkdirAll(targetPath, 0755)
```

Comment:
I hit the same problem, fixed in https://github.com/elastic/elastic-agent/pull/1058